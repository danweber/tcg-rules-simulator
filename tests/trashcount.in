VERSION:2
#[When Attacking] If you have 20 or more cards in your trash, unsuspend this monster
FIELD1:Attacker1,DW1-Trasher Attacker2,DW1-Trasher
# 19 cards
TRASH1:DW1-02 DW1-02 DW1-02 DW1-02 DW1-02 DW1-02 DW1-02 DW1-02 DW1-02 DW1-02 DW1-02 DW1-02 DW1-02 DW1-02 DW1-02 DW1-02 DW1-02 DW1-02 DW1-02
FIELD2:Target,DW1-03
GAME:P1 T3 MAIN 3
SECURITY2:DW1-03 DW1-03 DW1-03
DUMP:DP

ECHO: -- attack, only 19 in trash
CMD:ATTACK Attacker1 Player
DUMP

ECHO: -- attack, now 20 in trash
CMD:ATTACK Attacker2 Player
DUMP

EXIT
