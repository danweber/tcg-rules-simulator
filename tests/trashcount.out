FIELD1:Attacker1,DW1-Trasher,1000DP Attacker2,DW1-Trasher,1000DP
FIELD2:Target,DW1-Warpredmonster,2000DP
GAME:P1 T3 MAIN 3
ECHO: -- attack, only 19 in trash
FIELD1:Attacker2,DW1-Trasher
FIELD2:Target,DW1-Warpredmonster
GAME:P1 T3 MAIN 3
ECHO: -- attack, now 20 in trash
FIELD1:
FIELD2:Target,DW1-Warpredmonster,REST
GAME:P1 T3 MAIN 3
